docs/uml-class-diagrams/admin/test/SatelliteTests/SatelliteTests.puml

```diff
@@ -2,7 +2,14 @@
 
 package edu.oakland.test.admin {
     class SatelliteTests {
-        
+       + void satNameInIsSatNameOut()
+       + void strengthInIsStrengthOut()
+       + void locInIsLocOut()
+       + void setStrengthIsStrength()
+       + void newLatPointUsedIncrementLat()
+       + void newLngPointUsedIncrementLng()
+       + void newTimeIsIncrementedByHour()
+       + void strengthIsBtwnOneAndTen()    
```

Comment:
I would add a few more tests for "satNameIsNotNull", "initLocationIsNotNull" and add a case for strengthInIsStrengthOut for the constructor and the setter, and strengthIsBtwnOneAndTen for the constructor and the setter.